Pearson-Advance/openedx-themes
# PADV-211: Update pearson-aws-theme

## Ticket

https://agile-jira.pearson.com/browse/PADV-221

## Description
 
This PR updates the **pearson-aws-theme** logo and registration/login page.

## Screenshots

![Login Page](https://user-images.githubusercontent.com/8495728/193979353-104146db-c56d-4bac-8a9c-e08ec42d8cbc.png)
![Register Page](https://user-images.githubusercontent.com/8495728/193979362-a798d79a-664f-4f39-be39-c8fce17b1025.png)

## Type of Change

- [x] Update pearson-aws-theme logo.
- [x] Update pearson-aws-theme registration/login form.

## Testing:

- Check out this branch.
- Enable Comprehensive theming: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html
- Compile assets: `paver update_assets --themes=pearson-aws-theme`
- Assign **pearson-aws-theme** to a new site.

## Reviewers
